src/ivector/plda.h

```diff
@@ -144,7 +155,8 @@ class Plda {
   Vector<double> offset_;  // derived variable: -1.0 * transform_ * mean_
 
  private:
-  KALDI_DISALLOW_COPY_AND_ASSIGN(Plda);
+  Plda &operator = (const Plda &other);  // disallow assignment
```

Comment:
Does this need to be changed?

Reply:
Yes, since we need to use the PLDA copy constructor elsewhere. Now, only assignment is disallowed.